packages/devtools_app/lib/src/memory/memory_tracker_model.dart

```diff
@@ -112,7 +112,7 @@ class _TrackerCountColumn extends ColumnData<Tracker> {
   }
 
   @override
-  double get fixedWidthPx => 50.0;
+  double get fixedWidthPx => 300.0;
```

Comment:
This should match the above constructor 100  value.  If set to 100 then the width is wide enough.